### PR TITLE
Initial implementation for replica set detail view

### PR DESCRIPTION
**Pull request by bryk**
The implementation consists of backend code and sample frontend view
that dumps the data into an ugly looking table. Subsequent changes will
implement frontend so that things look pretty.

### CODE REVIEW

**Comment by bryk** [comment]
@floreks Can you review?

**Comment by floreks** [comment]
Sure, I will do it tomorrow morning.

File: src/app/frontend/replicasetdetail/replicasetdetail.html [hunk 1]
```diff
@@ -14,7 +14,27 @@
...
+      <tbody>
+        <tr ng-repeat="pod in ctrl.replicaSetDetail.pods">
+          <td>{{pod.name}}</td>
+          <td>{{pod.startTime}}</td>
+          <td>{{pod.podIP}}</td>
+          <td>{{pod.nodeName}}</td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
```
Comment (floreks) [inline src/app/frontend/replicasetdetail/replicasetdetail.html]: For small pieces of data this table is good but i think that for future references and bigger clusters some kind of md-data-table like here: [http://danielnagy.me/md-data-table/] would be good to have (searching, paging, etc.). Just something to have in mind.

Comment (bryk) [inline src/app/frontend/replicasetdetail/replicasetdetail.html]: Yes, agreed. I just dumped the detail information here in this pull request. I didn't want to make the PR too long. In next change I'll actually implement the UI to make it look pretty. Most likely using angular material constructs.

File: src/app/frontend/replicasetdetail/replicasetdetail_state.js [hunk 2]
```diff
@@ -52,5 +52,21 @@ export default function stateConfig($stateProvider) {
...
+ * @param {!StateParams} $stateParams
+ * @param {!angular.$resource} $resource
+ * @ngInject
+ */
+function resolveDetails($stateParams, $resource) {
+  /** @type {!angular.Resource<!backendApi.ReplicaSetDetail>} */
+  let resource = $resource('/api/replicasets/:namespace/:replicaSet', $stateParams);
+
+  return resource.get().$promise;
+}
```
Comment (floreks) [inline src/app/frontend/replicasetdetail/replicasetdetail_state.js]: @maciaszczykm has already created PR with external resolver file for this.
https://github.com/maciaszczykm/dashboard/commit/7fe92359ec53e61afc2e274259f516234551997b
Maybe this PR can be merged first and then this function can be moved.

**Comment by floreks** [comment]
LGTM :)

File: src/app/frontend/replicasetdetail/replicasetdetail_state.js (hunk 2 above)
Comment (bryk) [inline src/app/frontend/replicasetdetail/replicasetdetail_state.js]: Ok, I'll merge this and then you rebase. Thanks!

**Pull request merged**